### ISSUE TITLE
Create Recent Episodes component

### ISSUE THREAD

**Issue by jroebu14** [BBC-archive/psammead#3828]
**Is your feature request related to a problem? Please describe.**
We need a component to display recent onDemand radio episodes.

**Describe the solution you'd like**

Each episode displayed as an item in the component should include the following information:
- Play icon  (this should be hidden from screenreaders)
- Brand Title e.g. "Le Journal" 
- An Episode Title if it exists
- Release Date of the episode as datestamp e.g. "4 July 2020"
- Duration of the programme (in the service numerals where applicable) e.g. "Duration 59

A team accessiblity swarm should be carried out checking the usual a11y checks and accessiblity acceptance criteria below.

### Designs
There will be 2 variations of how the items will look based on whether an episode title is available. The MVP for this component is the variation without an episode title - consider a separate PR for the work with an episode title.

**without episode title**
<img width="250" alt="Screenshot 2020-10-05 at 15 39 29" src="https://user-images.githubusercontent.com/4798332/95093799-11dddd00-0721-11eb-97f2-d0edaafdee0c.png">

**with episode title**
![Screenshot 2020-10-08 at 14 45 03](https://user-images.githubusercontent.com/22519682/95467026-e2bfa980-0974-11eb-887a-4985c928d1b4.png)

**unordered list with 4 items**
<img width="591" alt="Screenshot 2020-10-05 at 15 16 01" src="https://user-images.githubusercontent.com/4798332/95092249-536d8880-071f-11eb-8a6c-b59134ca2f3c.png">

<img width="909" alt="Screenshot 2020-10-05 at 15 13 55" src="https://user-images.githubusercontent.com/4798332/95090701-7ac35600-071d-11eb-8f75-51c802796674.png">

<img width="1264" alt="Screenshot 2020-10-05 at 15 13 47" src="https://user-images.githubusercontent.com/4798332/95090707-7c8d1980-071d-11eb-84b9-1c855acfc8b9.png">

<img width="1053" alt="Screenshot 2020-10-05 at 15 13 34" src="https://user-images.githubusercontent.com/4798332/95090712-7dbe4680-071d-11eb-8f8e-3f9a06c93267.png">

### HTML structure

With title 
- list item must wrap the anchor link, brand title, episode title and date
- Use CSS to make anchor wrap the whole list item so that the full item is clickable (example can be found in other Psammead promo components)

Here is how the jsx may look:
```jsx
<li>
    <a>
      <span>{brandtitle}</span>
      <span>{episodetitle || date}</span>
      {episodetitle ? <span>date</span> : null}
    </a>
  <span>{duration}</span>
</li>
```

### a11y
Refer to https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5f0c3fc85b45918c4c1e8b46 for more info on a11y A/C

**Recent Episodes - Landmark**
Given I use a screen reader
When I navigate to ‘Recent episodes’ 
Then a complementary landmark will be announced
And this will not be nested within another landmark e.g. the main landmark
And this will be announced with a label e.g. "Recent episodes" in the language of the service
And the label will be unique to the page

**Recent Episodes - h2**
Given I use a screen reader
When I navigate to the ‘Recent episodes’ promo 
Then a heading level 2 will be announced as the first piece of content
And the h2’s announced text will be the same as the complementary landmark’s label for this section
And the h2 will be unique to the page
And the section will not include any other h2’s

**Recent Episodes - unordered List** 
Given I use a screen reader
And I am viewing the ‘Recent episodes’ component
When there is just one promo 
Then it should not be announced as a list item

Given I use a screen reader
And I am viewing the ‘Recent episodes’ component
When there is more than one promo 
Then the first promo should be announced as a list item within a list
 
Given I am using a screenreader
When I navigate to a ‘Recent episodes’ promo
Then this should not contain heading semantics

Given I am viewing the ‘Recent episodes’ component
When I change my colour preferences in Firefox
And when I am viewing in a custom high contrast mode on PC
Then I should be able to see the divider lines between promos
And the media indicator icon (default, hover and focus states)

**Recent Episodes without Episode Title** 
Given there is no episode title in the recent episode promo data 
and I am using a screenreader
When I navigate to the link in a ‘Recent episodes’ promo
Then the announced link text is “Audio, [Brand Title], [Release Date], Duration [duration]” in the language of the service
And the media indicator is not announced on it’s own 
And there are no heading semantics

**Recent Episodes with Episode Title**
Given there is episode title in the recent episode promo for an episode
and I am using a screenreader
When I navigate to the link for that episode in the ‘Recent episodes’ promo 
Then the announced link text is “Audio, [Brand Title], [Episode title], Duration [duration]”  in the language of the service
and when the user navigates to the release date 
then the release date is announced to the user
And the media indicator is not announced on it’s own
And there are no heading semantics

**Links** 
Given I use a keyboard only
When I navigate to a promo by pressing the TAB key
Then only links must be focusable
And these must have visible state change on focus 
And this must not be identified by colour alone
And be in addition to the browser default focus outline styling
 

Given I use a mouse
When I hover over any part of a promo
Then the mouse cursor will change from a pointer to a hand
And the link must have a visible state change
And this must not be identified by colour alone

**NB**
Consider UX for both RTL and LTR services
Add new component to Storybook

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**
Jira - https://jira.dev.bbc.co.uk/browse/WSMEDIA-22
Zeplin - https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/dashboard?seid=5f0c400a84a1689e6049b0cd

**Comment by aeroplanejane** [comment]
@rebeccamcginn @ryanmccombe we're going to keep these as a standard links and NOT h3s 
This is for a11y purposes (a h3 implies there is content beneath it but in this case there is not, they are just standard links) 
cc @greenc05

**Comment by rebeccamcginn** [comment]
Closed by automation
